index.html

```diff
@@ -8,11 +8,12 @@
     <meta name="description" content="Oppia Foundation">
 
     <link rel="shortcut icon" type="image/png" href="static/images/favicon.png">
-    <link rel="stylesheet" type="text/css" media="screen" href="static/css/stylesheet.css">
-    <link rel="stylesheet" type="text/css" media="screen" href="third_party/css/devices.min.css">
 
+    <link rel="stylesheet" type="text/css" media="screen" href="third_party/css/devices.min.css">
     <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/angular_material/1.1.9/angular-material.min.css">
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Any reason to use v3 versus v4 (which is the newest version, I think)?

(If there's a reason to use the older version, it might be worth documenting that as a comment.)